iphone/Maps/UI/Help/AboutController/Models/AboutInfo.swift

```diff
@@ -0,0 +1,74 @@
+enum AboutInfo: CaseIterable {
```

Comment:
Why CaseIterable is needed?

Reply:
Thanks! I used this one to get allCases previously.